cyences_app_for_splunk/appserver/static/cs_overview.js

```diff
@@ -56,7 +56,7 @@ require([
     ]
 
     let panel_depends_tokens = [
```

Comment:
I think this list we don't have to change as it used in show/hide overview dashboard panel based on enable/disable products.

so no need to add Authentication, Ransomware and Sysmon tags here.

Reply:
Let's keep them I think just as fail safe.

I think Sysmon is needed as Sysmon we have it as product in the product list.